commands/config/init.go

```diff
@@ -39,8 +39,14 @@ func initInitCommand() *cobra.Command {
 		Args: cobra.NoArgs,
 		Run:  runInitCommand,
 	}
-	initCommand.Flags().BoolVar(&initFlags._default, "default", false, "If omitted, ask questions to the user about setting configuration properties, otherwise use default configuration.")
-	initCommand.Flags().StringVar(&initFlags.location, "save-as", "", "Sets where to save the configuration file [default is ./.cli-config.yml].")
+	initCommand.Flags().BoolVar(&initFlags._default,
+		"default",
+		false,
+		"If omitted, ask questions to the user about setting configuration properties, otherwise use default configuration.")
+	initCommand.Flags().StringVar(&initFlags.location,
+		"save-as",
+		"",
+		"Sets where to save the configuration file [default is ./.cli-config.yml].")
 	return initCommand
```

Comment:
All these newline changes makes the commands `Flags()` initialization different with 3 different styles across the codebase:

STYLE 1 (all in one line):
```go
	command.Flags().StringVar(&flags.vidPid, "vid-pid", "", "When specified, VID/PID specific build properties are used, if boards supports them.")
```

STYLE 2 (break on the description):
```go
	command.Flags().StringVar(&flags.vidPid, "vid-pid", "",
		"When specified, VID/PID specific build properties are used, if boards supports them.")
```

STYLE 3 (break on all parameters):
```go
	initCommand.Flags().BoolVar(&initFlags._default,
￼ 		"default",
￼ 		false,
￼ 		"If omitted, ask questions to the user about setting configuration properties, otherwise use default configuration.")
```

IMHO we should change all of them, in the `commands/xxxx` modules (maybe to STYLE 2 since the line break is likely to happen on the description).

Reply:
Done 191cfc37d955d11e4700e8b1007e0dfa51ebc171